keps/sig-auth/3257-trust-anchor-sets/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: Cluster Trust Bundles
+kep-number: 3257
+authors:
+  - "@ahmedtd"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-auth
+status: provisional
```

Comment:
Update to `implementable` if this is still targeting v1.26 🤞

Reply:
We have a last-minute scope reduction.  I'm working on a doc update now.  I'll incorporate the status change.

Reply:
That update has been pushed